cmds/modules/provisiond/reporter.go

```diff
@@ -166,87 +166,16 @@ func (r *Reporter) getVmMetrics(ctx context.Context, slot rrd.Slot) error {
 	return nil
 }
 
-// getNetworkMetrics will collect network consumption for network resource and store it in the given slot
-func (r *Reporter) getNetworkMetrics(ctx context.Context, slot rrd.Slot) error {
-	log.Debug().Msg("collecting networking metrics")
-	stub := stubs.NewNetworkerStub(r.cl)
-
-	ctx, cancel := context.WithTimeout(ctx, 1*time.Minute)
-	defer cancel()
-	metrics, err := stub.Metrics(ctx)
-	if err != nil {
-		return err
-	}
-
-	for wl, consumption := range metrics {
-		nu := consumption.Nu()
-		log.Debug().Str("network", wl).Uint64("computed", uint64(nu)).Msgf("consumption: %+v", consumption)
-		if err := slot.Counter(wl, float64(nu)); err != nil {
-			return errors.Wrapf(err, "failed to store metrics for '%s'", wl)
-		}
-	}
-
-	return nil
-}
-
-// getVmMetrics will collect network consumption every 5 min and store
-// it in the rrd database.
-func (r *Reporter) getGwMetrics(ctx context.Context, slot rrd.Slot) error {
```

Comment:
This will probably be re-added

Reply:
we decided to postpone gw and qsfs for later versions of zos light